Source/Libraries/Common/RTCVCommon.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+  <PropertyGroup>
+    <TargetFramework>net471</TargetFramework>
+    <OutputType>Library</OutputType>
+    <RootNamespace>RTCV.Common</RootNamespace>
+    <AssemblyName>RTCV.Common</AssemblyName>
```

Comment:
This name still collides with: https://github.com/redscientistlabs/RTCV/blob/c343682be18ec3366330666f5162b6202bd5b11d/Source/Libraries/Common/Common.csproj#L11

If the intention is to build a "RTCV.CommonLite" library, that has a subset, of the "RTCV.Common" code, then we should choose a different name here.

Reply:
Good idea.